bundles/org.openhab.binding.solarwatt/README.md

```diff
@@ -0,0 +1,154 @@
+|fractionCPULoadAverageLastFiveMinutes | Number:Percent | Average 5 minute CPU load |
+|fractionCPULoadAverageLastFifteenMinutes | Number:Percent | Average 15 minute CPU load |
+
+### PVPlant
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+| powerACOut | Number:Energy | Energy produced by the PV in watts |
+| workACOut | Number:Energy | Energy produced by the PV in watt hours |
+
+### Location
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+| powerBuffered | Number:Energy | Energy flow into the storage system
+| powerSelfConsumed | Number:Energy | Energy consumed direct from PV plus energy stored
+| powerSelfSupplied | Number:Energy | Energy consumed direct from PV plus energy consumed from storage
+| powerConsumedFromGrid | Number:Energy | Energy consumed from the grid  
+| powerConsumedFromStorage | Number:Energy | Energy consumed from storage
+| powerConsumed | Number:Energy | Total energy consumed. All inner and outer consumers.
+| powerProduced | Number:Energy | Energy produced by the PV
+| powerOut | Number:Energy | Energy delivered to the grid  
+| powerDirectConsumed | Number:Energy | Energy consumed directly without energy put into storage or taken from storage
+| workBuffered | Number:Energy | Energy flow into the storage system
+| workSelfConsumed | Number:Energy | Energy consumed direct from PV plus energy stored
+| workSelfSupplied | Number:Energy | Energy consumed direct from PV plus energy consumed from storage
+| workConsumedFromGrid | Number:Energy | Energy consumed from the grid  
+| workConsumedFromStorage | Number:Energy | Energy consumed from storage
+| workConsumed | Number:Energy | Total energy consumed. All inner and outer consumers.
+| workProduced | Number:Energy | Energy produced by the PV
+| workOut | Number:Energy | Energy delivered to the grid  
+| workDirectConsumed | Number:Energy | Energy consumed directly without energy put into storage or taken from storage
+
+### PowerMeter, S0Counter, MyReservePowerMeter
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+| channelDirectionMetering | String | Representing which energy flow directions are metered. One off *IN*, *OUT*, *BIDIRECTIONAL* 
+| powerIn | Number:Energy | Energy metered flowing into the consumer
+| powerOut | Number:Energy | Energy metered flowing out of the producer  
+| workIn | Number:Energy | Energy metered flowing into the consumer
+| workOut | Number:Energy | Energy metered flowing out of the producer  
+| consumptionEnergySum | Number:Energy | Total energy in watt hours 
+
+### Inverter, MyReserveInverter, SunSpecInverter
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+| powerACOutMax | Number:Energy | Maximum power production 
+| powerACOutLimit | Number:Energy | Limit of power production
```

Comment:
These seem rather `Energy:Power`. Same for powerInstallledPeak .

Reply:
I couldn't find that in the CoreItemFactory.
```
    public static final String CALL = "Call";
    public static final String COLOR = "Color";
    public static final String CONTACT = "Contact";
    public static final String DATETIME = "DateTime";
    public static final String DIMMER = "Dimmer";
    public static final String IMAGE = "Image";
    public static final String LOCATION = "Location";
    public static final String NUMBER = "Number";
    public static final String PLAYER = "Player";
    public static final String ROLLERSHUTTER = "Rollershutter";
    public static final String STRING = "String";
    public static final String SWITCH = "Switch";
```

Reply:
Seemed a bit too late. I meant `Number:Power`.

Reply:
My mistake. They are already Number:Power but the README was off.